stories/example-screen-staff-list.stories.js

```diff
@@ -27,150 +17,1023 @@ export default {
+                <svg class="icon-lg-font-size mt-n1">
+                  <use xlink:href="${iconSprite}#tabler-adjustments-horizontal" />
+                  <span class="visually-hidden">Thumbs Down</span>
+                </svg>
+              </button>
+              <button class="avatar avatar-md avatar-image rounded-circle mt-2" type="button">
+                <img role="button" class="" src="https://www.edge.org/sites/default/files/styles/member-photo/public/member-pictures/picture-510-1416068804.jpg?itok=cnAQyDr5" alt="...">
+              </button>
+            </div>
+          </nav>
         </div>
-      </nav>
+      </div>
     </header>
-  `;
-};
+  </div>
+`
+
+const tabsDoubleRowHTML = `
+  <ul class="nav nav-tabs bg-white rounded-top border border-light ps-7 pe-7">
+    <li class="nav-item">
+      <a class="nav-link fs-5 active" aria-current="page" href="#">
+        <span>All Jobs</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Public</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Pending</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Processing</span>
+      </a>
+    </li>
+  </ul>
+  <ul class="nav nav-tabs bg-white border border-light border-top-0 ps-7 pe-7">
+    <li class="nav-item">
+      <a class="nav-link active fs-5" href="#">
+        <span>All Jobs</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" aria-current="page" href="#">
+        <span>Recommended Jobs</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>My Favorites</span>
+      </a>
+    </li>
+    <li class="nav-item">
+      <a class="nav-link fs-5" href="#">
+        <span>Applications</span>
+      </a>
+    </li>
+  </ul>
+`;
+
+const tableContent = `
+  <table class="table align-middle fs-5 table-borderless">
+    <thead class="border-bottom border-light text-nowrap">
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Actions</th>
+        <th scope="col">
+          <button class="d-flex border-0 bg-white fw-bold p-0">
+            Last Name
+            <svg class="icon-lg-font-size d-flex">
+              <use xlink:href="${iconSprite}#tabler-caret-up" />
+            </svg>
+            <span class="visually-hidden">Sorted ascending</span>
+          </button>
+        </th>
+        <th scope="col">First Name</th>
+        <th scope="col">Student ID</th>
+        <th scope="col">Major(s)</th>
+        <th scope="col">Email</th>
+        <th scope="col">Class Level</th>
+        <th scope="col">Graduation Date</th>
+        <th scope="col">Email Verified</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr class="border-bottom border-light">
```

Comment:
For the table data, you can create a loop to populate it like so:

`search.stories.js lines 261-281`
```
    /* Populate results list for testing overflow */

    const createListItems = document.querySelector('.list-unstyled');
    createListItems.innerHTML = `
      ${Array(20)
        .fill()
        .map(
          (item, i) => `
        <li>
          <a class="dropdown-item d-flex p-0 mb-2" href="#" id="item-${i + 1}">
            <span>
              <svg width="1rem" height="1rem" class="me-2 mt-n1">
                <use xlink:href="static/media/tabler-sprite.3d36b3c4.svg#tabler-search"></us
              </svg>
            </span>
            <span class="flex-fill">
              Chad Hampton Is A Very Loooooooooong Name - ${i + 1}
            </span>
          </a>
        </li>
      `
        )
        .join('')}
    `;
```